lib/WebService/Stripe.pm

```diff
@@ -70,6 +70,10 @@ method get_balance(:$headers) {
     return $self->get( "/v1/balance", {}, headers => $headers );
 }
 
+method get_balance_history(:$headers) {
```

Comment:
huh well reading through the api docs, I see what you mean about the transaction vs. history. it's an interesting misnomer on the stripe side it seems. I doubt I would've have tried for a singular of history, I would have just had `get_balance_transaction` and `get_balance_history`, which more matches what stripe did there.

but we can go ahead and correct their grammar too, I guess, heh